content/source/docs/enterprise/users-teams-organizations/organizations.html.md

```diff
@@ -4,14 +4,82 @@ page_title: "Organizations - Terraform Enterprise"
 sidebar_current: "docs-enterprise2-users-teams-organizations-organizations"
 ---
 
+[teams]: ./teams.html
+[users]: ./users.html
+
 # Organizations
 
-Organizations are a shared space for teams to collaborate on workspaces.
-Organization owners set which teams have read, write, and admin privileges on
-which workspaces.
+Organizations are a shared space for [teams][] to collaborate on workspaces in Terraform Enterprise (TFE).
+
+## Selecting Organizations
+
+On most pages within TFE, the top navigation bar displays the name of the selected organization.
+
+If you are a member of multiple organizations, you can click the organization name for a drop-down menu of all the organizations you can view. You can switch to another organization by clicking its name, or you can create a new organization.
+
+![screenshot: the organization switcher menu](./images/org-nav.png)
+
+## Creating Organizations
+
+Users can create new TFE organizations. You can navigate to the new organization page using the organization switcher menu; this page also displays automatically when the currently logged-in user does not belong to any organizations, like when a new user account is created.
+
+![screenshot: the new organization page](./images/org-new.png)
+
+To create a new organization, you must provide a unique name and a primary contact email address. Organization names can include numbers, letters, underscores (`_`), and hyphens (`-`).
+
+Once you have created an organization, you can add other [users][] by adding them to one or more [teams][].
+
+-> **Note:** On the SaaS version of TFE, any user can create a new organization. On private installs of TFE, the administrators can restrict this ability, so that only site admins can create organizations. See [Administration: General Settings](../private/admin/general.html#organization-creation) for more details.
+
+## Organization Settings
+
+-> **API:** See the [Organizations API](../api/organizations.html).
+
+You can view and manage an organization's settings by clicking the main "Settings" link in the top navigation bar.
```

Comment:
I'd probably delete main here, it doesn't really provide any additional distinction.